src/table/cells/handle-cell-focus.js

```diff
@@ -0,0 +1,18 @@
+export const focusCell = (rowElements, nextCellCoord) => {
+  const nextCell = rowElements[nextCellCoord[0]].getElementsByClassName('sn-table-cell')[nextCellCoord[1]];
+  nextCell.focus();
+  nextCell.setAttribute('tabIndex', '0');
+};
+
+const resetTabIndex = (rowElements, focusedCellCoord) => {
+  const cell = rowElements[focusedCellCoord[0]]?.getElementsByClassName('sn-table-cell')[focusedCellCoord[1]];
+  cell && cell.setAttribute('tabIndex', '-1');
+};
+
+export const handleCellFocus = (cell, focusedCellCoord, rootElement) => {
+  const { rawRowIdx, rawColIdx } = cell;
+  const rowElements = rootElement.getElementsByClassName('sn-table-row');
+  resetTabIndex(rowElements, focusedCellCoord.current);
+  focusedCellCoord.current = [rawRowIdx + 1, rawColIdx];
```

Comment:
Why are you sett ing `.current`?

Reply:
that is decided by react https://reactjs.org/docs/refs-and-the-dom.html#accessing-refs https://reactjs.org/docs/hooks-reference.html#useref

Reply:
Ahh smart